lbrynet/lbrynet_daemon/Daemon.py

```diff
@@ -58,7 +45,7 @@
 from lbrynet.lbrynet_daemon.auth.server import AuthJSONRPCServer
 
 from lbrynet.metadata.Metadata import Metadata, verify_name_characters
-from lbrynet.core import log_support
+from lbrynet.core import support
```

Comment:
this should be log_support.

tests failed: https://travis-ci.org/lbryio/lbry/jobs/175889369#L495